### ISSUE TITLE
[WPF] "Point is empty" exception on MapControl.Info event 

### ISSUE THREAD

**Issue by IldarKhayrutdinov** [Mapsui/Mapsui#1276]
Windows 10 Pro
Visual Studio Community 2019 16.11.5
.NET Core 3.1
Mapsui.Wpf  3.0.1

Test project: [MapsuiTest.zip](https://github.com/Mapsui/Mapsui/files/7357124/MapsuiTest.zip)
Steps to reproduce:
1. Set breakpoint in `MapControlOnInfo` method
2. Run in debug mode
3. Left mouse click on map
4. Continue running (F5)
5. Press `LeftCtrl` key while moving the mouse
6. Exception:
![image](https://user-images.githubusercontent.com/29858598/137579098-9e9144a3-1e44-496a-8a08-38d4e2fe0794.png)

```
System.Exception
  HResult=0x80131500
  Message = Point is empty
  Source = Mapsui.Geometries
  StackTrace:
   at Mapsui.Geometries.Point.get_X()
   at Mapsui.UI.Wpf.MapControl.DrawBbox(Point newPos)
   at Mapsui.UI.Wpf.MapControl.MapControlMouseMove(Object sender, MouseEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.Run()
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run()
   at MapsuiTest.App.Main()

```

**Comment by pauldendulk** [comment]
Thanks for your very clear bug report. This is a peculiar situation, not easy to find, thanks for figuring this out. What happens is that because of the breakpoint the MouseUp event is not triggered and some of the fields are not reset. Without debug mode this will not occur, so it is not a very serious bug fortunately.

**Comment by pauldendulk** [comment]
I will fix by checking if the begin and endpoint are empty.

**Comment by pauldendulk** [comment]
I pushed a fix to develop/3.0. Since this only affects developers I will hold out a patch release until other bug come up.

**Comment by IldarKhayrutdinov** [comment]
@pauldendulk Thanks for developing Mapsui! I'm happy to be helpful
